library/std/src/personality/gcc.rs

```diff
@@ -77,6 +77,9 @@ const UNWIND_DATA_REG: (i32, i32) = (0, 1); // R0, R1
 #[cfg(any(target_arch = "riscv64", target_arch = "riscv32"))]
 const UNWIND_DATA_REG: (i32, i32) = (10, 11); // x10, x11
 
+#[cfg(target_arch = "loongarch64")]
+const UNWIND_DATA_REG: (i32, i32) = (4, 5); // A0, A1
```

Comment:
```suggestion
const UNWIND_DATA_REG: (i32, i32) = (4, 5); // a0, a1
```

nit

Reply:
All architectures use uppercase except riscv, I think we should refer to most of the architecture